WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -297,7 +297,10 @@ class NotificationsViewController: UIViewController, UIViewControllerRestoration
 
     static func viewController(withRestorationIdentifierPath identifierComponents: [String],
                                coder: NSCoder) -> UIViewController? {
-        return WPTabBarController.sharedInstance().notificationsViewController
+        guard let tabBarController = RootViewControllerCoordinator.sharedPresenter.rootViewController as? WPTabBarController else {
+            return nil
+        }
+        return tabBarController.notificationsViewController
```

Comment:
Same suggestion as with `readerCoordinator`, for the sake of consistency I think it makes sense to move it to `RootViewPresenter` interface. 

We'll clearly see the differences between the 2 implementations, additionally, we could track when the app tries to access the variable unexpectedly, same way we track in the `fallback` methods.

Reply:
Done in f3f2794f217e75a2c479eb98b7e77ccca5276c60. And tracking is added in c17f29e324079c82b23da1b0f1fc28c100fd41fc 👍